css/index.css

```diff
@@ -1,3 +1,157 @@
 /* Example of @import use
 This imports the `reset.css` file */
-@import "./reset.css";
+@import url("./reset.css");
+
+/**
+*? Styles for the Body.
+**/
+
+body {
+  background: #191765;
+  min-width: 375px;
+  padding-top: 24px;
+  padding-bottom: 24px;
+}
+
+/**
+*? The circle background color style.
+**/
+
+.circle-bg-box {
+  min-width: 1300px;
+  height: 535px;
+  left: 9px;
+  top: 37px;
+  position: absolute;
```

Comment:
There are too many instances of repetitions of classes... these classes can be optimized by using CSS variables or combining the classes like below...

```css
.circle-bg-box,
.circle-bg-1,
.circle-bg-2 {
  position: absolute;
}
```

You should do this for all occurrences

Another thing you can do is to have a utility class and reuse them where possible in your HTML

```css
.absolute{
  position: absolute;
}
```

```html
<div class="absolute circle-bg-box">Hello world!</div>
```

Reply:
### I really appreciate this sir, the repeated classes will be modified  .